### ISSUE TITLE
Tabs supported ?

### ISSUE THREAD

**Issue by x9t9** [picturepan2/spectre#186]
Is the [Tabs component](https://picturepan2.github.io/spectre/components.html#tabs) fully supported ? 
In fact , it does not even apepar on the sidebar menu in the docs website , and even in the repo readme it is not mentioned .
The reason that I ask, is that , in face , it is not working for me ( Tested Chrome ,Firefox , Vivaldi ).
At any case, if it is fully supported , need to update the docs ?

**Comment by picturepan2** [comment]
@x9t9 It is not pure CSS Tabs component, only the style code right now. Tabs, Breadcrumbs, Pagination are all included in [Navigation](https://picturepan2.github.io/spectre/components.html#navigation).

**Comment by x9t9** [comment]
@picturepan2 - thanks . So One should handle the tabs ( css , JS ) by oneself , right ?
Great job BTW on spectre ..

**Comment by picturepan2** [comment]
@x9t9 Yes. Or you can use `input radio` pure CSS trick. Demo here: https://plnkr.co/edit/z45r86J0VfpWMhII5CKu?p=preview .

**Comment by AGhasemzadeh** [comment]
I think it would be good idea to have in built-in.

**Comment by x9t9** [comment]
Is there any progress on this in new releases ?

**Comment by niutech** [comment]
Pure CSS tabs are implemented in my fork: https://niutech.github.io/spectre/components/tabs.html
